config/translator.php

```diff
@@ -31,6 +31,14 @@
             'model' => env('OPENAI_MODEL', 'gpt-4o'),
             'api_key' => env('OPENAI_API_KEY'),
             'organization_id' => env('OPENAI_ORGANIZATION'),
+
+            /**
+             * Custom prompt to enhance translation quality.
+             * 
+             * Example:
+             * 'custom_prompt' => 'This application translates medical terms consistently throughout.'
+             */
+            'custom_prompt' => ''
```

Comment:
`prompt_extension` perhaps? custom_prompt I would assume it replaced the entire prompt.

Reply:
yes, I think this is better

Reply:
or maybe additional_prompt?

Reply:
Definitely better could also consider `prompt_context`.

Reply:
I have added now, you can think about better name for this